daemon-tests/tests/happy_path.rs

```diff
@@ -324,6 +324,37 @@ async fn maker_rejects_rollover_of_open_cfd() {
     wait_next_state!(order_id, maker, taker, CfdState::Open);
 }
 
+#[tokio::test]
+async fn maker_rejects_rollover_after_commit_finality() {
+    let _guard = init_tracing();
+    let oracle_data = OliviaData::example_0();
+    let (mut maker, mut taker, order_id) =
+        start_from_open_cfd_state(oracle_data.announcement()).await;
+
+    taker.mocks.mock_latest_quote(Some(dummy_quote())).await;
+    maker.mocks.mock_latest_quote(Some(dummy_quote())).await;
+    next_with(taker.quote_feed(), |q| q).await.unwrap(); // if quote is available on feed, it propagated through the system
+
+    taker.trigger_rollover(order_id).await;
+
+    wait_next_state!(
+        order_id,
+        maker,
+        taker,
+        CfdState::IncomingRolloverProposal,
+        CfdState::OutgoingRolloverProposal
+    );
+
+    deliver_event!(maker, taker, Event::CommitFinality(order_id));
+    // If it wasn't for the collab settlement, it would be "OpenCommitted"
```

Comment:
What does this comment refer to?

Reply:
oops, I started with the other protocol, then changed my mind, and forgot to update the comment.... fixed!